commodore/lib/commodore.libjsonnet

```diff
@@ -123,6 +123,16 @@ local envList(map) = [
   for x in std.objectFields(map)
 ];
 
+/*
+ * \brief Helper to get a fields value or a default.
+ *
+ * \arg dict An arbitrary dictionary data structure.
+ * \arg field The name of a field.
+ * \arg default The value to return if the field is not present on the dict.
+ */
+local getValueOrDefault = function(dict, field, default)
```

Comment:
Consider using the following style which we use for other functions with arguments in this file:

```suggestion
local getValueOrDefault(dict, field, default) =
```